sdk/core/azure-core-test/src/test_proxy_manager.cpp

```diff
@@ -243,6 +243,8 @@ void TestProxyManager::SetProxySanitizer()
   addSanitizer(SanitizerType::General, g_accountRegex, "account");
   addSanitizer(SanitizerType::Body, "client_secret=(?<clientsecret>[^&]+)", "clientsecret");
   addSanitizer(SanitizerType::Body, "client_id=(?<clientid>[^&]+)", "clientid");
+  addSanitizer(SanitizerType::Body, "(?:<Value>)(.*)(?:</Value>)", "value");
```

Comment:
What's the `Value` here, typically, and is it sensitive info? This one seems pretty generic to redact.

Reply:
it's actually an xml encoded thng the regex says look for Value.../Value

Reply:
show only in a handful of storage recordings

Reply:
FWIW, that should only affect SDKs whose bodies are encoded as XML, which I *think* only affects Storage.

Reply:
From @alzimmermsft
> Okay, they do use \<Value>x\</Value> elsewhere when the value isn't a secret in Blob Tags, that regex will need a look around for \<UserDelegationKey\>

Reply:
digging a bit deeper into this value in userdelegationkey in the case of those combos the value is already redacted 
E.G. \u003CValue\u003E**REDACTED**\u003C/Value\u003E\u003C/UserDelegationKey